internal/namespaces/instance/v1/custom_server.go

```diff
@@ -949,11 +949,13 @@ func serverTerminateCommand() *core.Command {
 
 			var multiErr error
 			if terminateServerArgs.WithIP && server.Server.PublicIP != nil && !server.Server.PublicIP.Dynamic {
+				fmt.Printf("delete IP...\n")
```

Comment:
Can you use `interactive.Print` instead of `fmt.Printf`. Using printf like this would break command like `scw xxxx | jq`

Reply:
Whoops, that was a debug statement which was accidentally committed 🤦 

I'll delete it, to match the behaviour of the delete command